collections-csharp/MergingDictionariesInCSharp/MergingDictionariesInCSharp/Methods/UsingHashSetMethod.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace MergingDictionariesInCSharp
+{
+    public partial class Methods
+    {
+        public static Dictionary<TKey, TValue> UsingHashSetMethod<TKey, TValue>(params Dictionary<TKey, TValue>[] dictionaries)
```

Comment:
This method will not work as expected. It is working with the test cases that are being thrown at it, but the basis of the method is expecting that both HashSets are storing the keys and values in the same order, but this is explicitly not guaranteed by HashSet.

Reply:
You are correct Jeff. HAshet collection does not guarantee that the items are stored in order we add them. I changed it to List instead of HashSet.